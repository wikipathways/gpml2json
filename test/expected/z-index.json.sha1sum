735070bc4337066a52b433fe804f3d524e993226 ?-
