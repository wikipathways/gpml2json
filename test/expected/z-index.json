{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["fd2a7","f017c","d1115","ef0d5","b47f2","d2d79","c8e4a","d9e67","cd14e","c43e6","f9f51","c77d0"],"drawAs":"rect","gpmlElementName":"Pathway","height":526.1553615404016,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":472.4153917921197,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130807","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/3e988a8f879aa9e6258fb634a3dcc63b71998e94905d69323b6bc850390c1b80"},"entityMap":{"fd2a7":{"id":"fd2a7","x":26.590196802781833,"y":148.58287895691657,"backgroundColor":"#993300","fontSize":10,"height":46.75550660792965,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":106.22687224669693,"zIndex":32965,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c8e4a":{"id":"c8e4a","textContent":"Label\nFill: white\nStroke width: 0\nz-index: in front","x":89.82632984305803,"y":370.66518936848877,"backgroundColor":"#ffffff","fontSize":12,"height":119.30921454152683,"borderWidth":0,"drawAs":"rect","verticalAlign":"middle","width":138.829874074335,"zIndex":32971,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"f017c":{"id":"f017c","x":30.981268286363814,"y":288.74346726728436,"backgroundColor":"#993300","fontSize":10,"height":46.75550660792965,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":106.22687224669693,"zIndex":32966,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"d9e67":{"id":"d9e67","textContent":"Label\nStroke: black\nFill: white\nz-index: in front","x":84.23161618667075,"y":236.21585016143965,"backgroundColor":"#ffffff","fontSize":12,"height":119.30921454152683,"drawAs":"rect","verticalAlign":"middle","width":138.829874074335,"zIndex":32972,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"d1115":{"id":"d1115","x":36.5759819427511,"y":423.1928064743327,"backgroundColor":"#993300","fontSize":10,"height":46.75550660792965,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":106.22687224669693,"zIndex":32967,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c77d0":{"id":"c77d0","textContent":"Label\nStroke: black\nFill: transparent\nz-index: in front","x":80.83954570208982,"y":94.05725985307063,"fontSize":12,"height":119.30921454152683,"drawAs":"rect","verticalAlign":"middle","width":138.829874074335,"zIndex":32976,"textAlign":"center","color":"#000000","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"},"ef0d5":{"id":"ef0d5","textContent":"z-index: behind","x":264.66208782736,"y":18.23025140825382,"color":"#cccccc","backgroundColor":"#0000ff","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32968,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b47f2":{"id":"b47f2","textContent":"z-index: behind","x":326.51230809167873,"y":238.6707800426044,"backgroundColor":"#ff6600","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32969,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"d2d79":{"id":"d2d79","textContent":"z-index: in front","x":269.15547989784613,"y":189.0232029500999,"color":"#cccccc","backgroundColor":"#0000ff","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32970,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"cd14e":{"id":"cd14e","textContent":"z-index: behind","x":311.51230809167873,"y":395.6707800426038,"backgroundColor":"#ff6600","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32973,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c43e6":{"id":"c43e6","textContent":"z-index: in front\nfill: transparent","x":254.15547989784628,"y":346.02320295009866,"color":"#999999","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32974,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f9f51":{"id":"f9f51","textContent":"z-index: in front","x":322.0189160211926,"y":67.87782850075905,"backgroundColor":"#ff6600","fontSize":10,"height":100.48458149779776,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":115.90308370044096,"zIndex":32975,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"}}}
